ci/scripts-aws-sdk-3/get-deployed-image-tags.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+
+const util = require('util')
+
+const { ECS, paginateListServices } = require('@aws-sdk/client-ecs')
+
+const ecsClient = new ECS()
+
+async function getImageTagsForTaskDef (service, taskDefinition) {
+  const taskDefResponse = await ecsClient.describeTaskDefinition({
+    taskDefinition: taskDefinition
+  })
+
+  return {
+    service: service,
+    containerImages: taskDefResponse.taskDefinition.containerDefinitions.map(
+      containerDef => {
+        // Images can either be repository:tag or repository@digest
+        const delimiter = containerDef.image.includes('@') ? '@' : ':'
+
+        return {
+          repository: containerDef.image.split(delimiter)[0],
+          tag: containerDef.image.split(delimiter)[1]
+        }
+      }
+    )
+  }
+}
+
+async function describeServices (services) {
+  const taskDefMap = new Map()
```

Comment:
This is more personal choice and left here as a musing, I prefer declaring variables close to where they're used rather than at the start of the scope as I feel it helps readability and refactoring later on, as such I would have put `const taskDefMap = new Map()` on line 47.